mysqli/mysqli.php

```diff
@@ -216,14 +217,15 @@ public function close () {}
 	public function commit ($flags = null, $name = null) {}
 
 	/**
-	 * @param $host [optional]
-	 * @param $user [optional]
-	 * @param $password [optional]
-	 * @param $database [optional]
-	 * @param $port [optional]
-	 * @param $socket [optional]
+	 * @link https://php.net/manual/en/function.mysqli-connect.php
+	 * @param string $host [optional]
+	 * @param string $user [optional]
+	 * @param string $password [optional]
+	 * @param string $database [optional]
+	 * @param int $port [optional]
+	 * @param string $socket [optional]
 	 */
-	public function connect ($host, $user, $password, $database, $port, $socket) {}
+	public function connect ($host = '', $user = '', $password = '', $database = '', $port = 0, $socket = '') {}
```

Comment:
According to https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L61 
Stub is
```
public function connect(
        ?string $host = null,
        ?string $user = null,
        ?string $password = null,
        ?string $database = null,
        ?int $port = null,
        ?string $socket = null
    ) {}
```
so I think default value should be changed to nulls